<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>casesMainView</class>
 <widget class="QDialog" name="casesMainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonEditCase">
     <property name="text">
      <string>Редактировать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonAddCase">
     <property name="text">
      <string>Добавить дело</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="buttonExit">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonDelete">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QTableView" name="casesView"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="buttonAddPayment">
     <property name="text">
      <string>Доб. платеж</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Дела в работе</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buttonSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="4">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="buttonReport">
     <property name="text">
      <string>Отчет (плат. / соб.) </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonDelPay">
     <property name="text">
      <string>Удалить платеж</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="buttonAllCases">
     <property name="text">
      <string>Все дела</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="buttonDocsCase">
     <property name="text">
      <string>К документам по делу</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
